### PR TITLE
Adds shared texture support for libgdx widgets

### PR DESCRIPTION
**Pull request by dattanand**
Adding shared texture support to be used with libgdx widgets

### CODE REVIEW

File: GVRf/Framework/src/org/gearvrf/GVRSharedTexture.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gearvrf;
+
+/**
```
Comment (liaxim) [inline GVRf/Framework/src/org/gearvrf/GVRSharedTexture.java]: You might want to update the javadoc to update the intended purpose of this class

**Comment by liaxim** [comment]
The shared_texture part looks good to me.

**Comment by liaxim** [comment]
I am not entirely sure about this but instead of having the SharedTexture class we could retrofit the base GVRTexture class to support the case SharedTexture exists for. It might be less confusing to have one type of texture less.

**Comment by dattanand** [comment]
I can think of creating gvrtexture base type object and passing texture ID, Kindly suggest how can it be achieved if we want to explicitly remove the term 'shared'

**Comment by dattanand** [comment]
tracked via #367

**Pull request closed without merging**